nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-stateless/README.md

```diff
@@ -38,19 +41,19 @@ Where the arguments dictate the runtime to use:
 
 ### Examples:
 ```
-1) docker run --rm -it nifi-stateless:1.10.0-SNAPSHOT-dockermaven \
-    RunFromRegistry Once --file /Users/nifi/nifi-stateless-configs/flow-abc.json
+1) ${NIFI_HOME}/bin/nifi.sh stateless RunFromRegistry Once --file /Users/nifi/nifi-stateless-configs/flow-abc.json
 2) docker run --rm -it nifi-stateless:1.10.0-SNAPSHOT-dockermaven \
+    RunFromRegistry Once --file /Users/nifi/nifi-stateless-configs/flow-abc.json
+3) docker run --rm -it nifi-stateless:1.10.0-SNAPSHOT-dockermaven \
     RunYARNServiceFromRegistry http://127.0.0.1:8088 nifi-stateless:latest kafka-to-solr 3 --file kafka-to-solr.json
-3) docker run -d nifi-stateless:1.10.0-SNAPSHOT-dockermaven \
+4) docker run -d nifi-stateless:1.10.0-SNAPSHOT-dockermaven \
```

Comment:
Should these docker run lines specify `apache/nifi-stateless`?

Reply:
The project builds the image as nifi-stateless:1.10.0-SNAPSHOT-dockermaven. apache/nifi-stateless is not available until it is explicitly tagged and pushed to docker hub (after a release). Once we have Stateless NiFi in the main docs, it should be apache/nifi-stateless for end users.

Reply:
Sorry, I misunderstood your comment. You are right.